data/tutorial/04_contents.md

```diff
@@ -1,28 +1,28 @@
 ---
 path: "/tutorial/contents"
-title: "Custom content types"
+title: "Custom Content Types"
 ---
 
-At some point working with `Irmin` you will probably want to move beyond using
+At some point while working with Irmin, you might want to move beyond using
 the default content types.
 
 This section will explain how custom datatypes can be implemented using
-[Irmin.Type][irmin.type]. Before continuing with these examples make sure to
-read through the [official documentation][irmin.type], which has information
+[`Irmin.Type`][irmin.type]. Before continuing with these examples, make sure to
+read through the [official documentation][irmin.type], as it contains information
 about the predefined types and how they're used.
```

Comment:
Should we put in a few of the most important ones, or do they need to read all that documentation to move forward?